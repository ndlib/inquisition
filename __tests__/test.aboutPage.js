import React from 'react'
import renderer from 'react-test-renderer'
import { useStaticQuery } from 'gatsby'
import AboutTemplate from '../src/templates/about'
import Store from '@ndlib/gatsby-theme-marble/src/store/ReduxWrapper'
import staticQuery from '../__mocks__/staticQueryHelper'

const testData = {
  markdownRemark: {
    html: 'html!',
    frontmatter: {
      title: 'title',
    },
  },
}

describe('about page', () => {
  it('renders correctly', () => {
    useStaticQuery.mockImplementation(() => staticQuery)

    const tree = renderer
      .create(<Store>
        <AboutTemplate location={{}} data={testData} />
      </Store>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
