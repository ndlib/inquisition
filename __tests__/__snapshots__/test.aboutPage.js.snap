// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about page renders correctly 1`] = `null`;
